simpleio.py

```diff
@@ -101,6 +102,59 @@ def shift_out(dataPin, clock, value, msb_first=True):
             tmpval = bool((value & (1 << i)))
             dataPin.value = tmpval
 
+class Servo:
+    """
+    Easy control for hobby (3-wire) servos
+
+    :param ~microcontroller.Pin pin: PWM pin where the servo is located.
+    :param int min_pulse: Minimum amount of microseconds allowed. Varies depending on type of servo.
+    :param int max_pulse: Maximum amount of microseconds allowed. Varies depending on type of servo.
```

Comment:
You are assuming that the servo ranges from 0 to 180 degrees correct? Why not just say that?